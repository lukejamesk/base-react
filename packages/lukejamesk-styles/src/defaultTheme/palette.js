const palette = {
  primary: '#39c5ff',
  secondary: '#5c946e',
  text: 'rgba(0, 0, 0, .9)'
}
export default palette
