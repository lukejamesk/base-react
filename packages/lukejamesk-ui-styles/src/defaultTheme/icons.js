const icons = {
  s: 12,
  m: 24,
  l: 48,
}

export default icons
