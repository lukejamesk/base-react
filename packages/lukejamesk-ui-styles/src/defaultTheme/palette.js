const palette = {
  backgroundColor: '#fff',
  primary: '#39c5ff',
  secondary: '#5c946e',
  light: '#efefef',
  borderLight: '#e6e6e6',
  dark: '#222',
  text: 'rgba(40, 40, 40, .9)',
}
export default palette
