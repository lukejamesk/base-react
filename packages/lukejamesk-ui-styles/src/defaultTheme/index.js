import palette from './palette'
import typography from './typography'
import spacing from './spacing'

const defaultTheme = {
  palette,
  typography,
  spacing
}

export default defaultTheme
