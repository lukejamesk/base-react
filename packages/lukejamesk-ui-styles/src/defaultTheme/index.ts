import palette, { Palette } from './palette'
export { Palette } from './palette'
import typography, { Typography } from './typography'
export { Typography } from './typography'
import spacing, { Spacing } from './spacing'
export { Spacing } from './spacing'
import breakpoints, { Breakpoints } from './breakpoints'
export { Breakpoints } from './breakpoints'
import icons, { Icons } from './icons'
export { Icons } from './icons'

export type Theme = {
  palette: Palette
  typography: Typography
  spacing: Spacing
  breakpoints: Breakpoints
  icons: Icons
}

const defaultTheme: Theme = {
  palette,
  typography,
  spacing,
  breakpoints,
  icons,
}

export default defaultTheme
