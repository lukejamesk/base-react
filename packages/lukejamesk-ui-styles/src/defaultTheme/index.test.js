import defaultTheme from '.'

describe('defaultTheme', () => {
  it('Snapshot', () => {
    expect(defaultTheme).toMatchSnapshot()
  })
})
