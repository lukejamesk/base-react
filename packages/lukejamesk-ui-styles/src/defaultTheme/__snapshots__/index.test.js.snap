// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultTheme Snapshot 1`] = `
Object {
  "breakpoints": Object {
    "l": 1280,
    "m": 768,
    "s": 480,
    "xl": 1920,
    "xs": 320,
    "xxl": 2560,
    "xxs": 0,
  },
  "icons": Object {
    "l": 48,
    "m": 24,
    "s": 12,
  },
  "palette": Object {
    "backgroundColor": "#fff",
    "borderLight": "#e6e6e6",
    "dark": "#222",
    "light": "#efefef",
    "primary": "#39c5ff",
    "secondary": "#5c946e",
    "text": "rgba(40, 40, 40, .9)",
  },
  "spacing": Object {
    "l": 20,
    "m": 16,
    "s": 12,
    "xl": 24,
    "xs": 8,
    "xxl": 28,
    "xxs": 4,
  },
  "typography": Object {
    "bold": 600,
    "button": Object {
      "fontFamily": "Verdana, Arial, Sans-serif",
      "fontSize": 14,
    },
    "fontFamily": "Helvetica, Arial, Sans-serif",
    "fontSize": 14,
    "heading1": Object {
      "fontFamily": "Verdana, Arial, Sans-serif",
      "fontSize": 25,
      "fontWeight": 600,
    },
    "heading2": Object {
      "fontFamily": "Verdana, Arial, Sans-serif",
      "fontSize": 21,
      "fontWeight": 600,
    },
    "heading3": Object {
      "fontFamily": "Verdana, Arial, Sans-serif",
      "fontSize": 18,
      "fontWeight": 600,
    },
  },
}
`;
